modules/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -314,7 +317,9 @@ param winRM object = {}
 ])
 param configurationProfile string = ''
 
-var vmComputerNameTransformed = vmComputerNamesTransformation == 'uppercase' ? toUpper(name) : (vmComputerNamesTransformation == 'lowercase' ? toLower(name) : name)
+var vmComputerName = !empty(vmCustomComputerName) ? vmCustomComputerName : name
```

Comment:
After assigning resource name as default, I'd suggest to remove this var
```suggestion
```

Reply:
Done